app/config/config.yml

```diff
@@ -26,7 +26,7 @@ framework:
     # See http://symfony.com/doc/current/reference/configuration/framework.html#ide
     ide: sublime
 
-    # esi:             ~
+    esi:             { enabled: true }
```

Comment:
@sergiu-popa, please add here a comment with a short description and a link to http://symfony.com/doc/current/book/http_cache.html#edge-side-includes